CruCentralCoast/Database/Event.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  Event.swift
+//  CruCentralCoast
+//
+//  Created by Cam Stocker on 5/2/18.
+//  Copyright © 2018 Landon Gerrits. All rights reserved.
+//
+
+import Foundation
+import Firebase
+
+class Event: NSObject, DatabaseObject {
+    var title: String
+    var startDate: Date
+    var endDate: Date?
+    var summary: String
+    var location: String?
+    var imageLink: String
+    @objc dynamic var image: UIImage?
+    
+    required init?(dict: NSDictionary) {
+        guard let title = dict["name"] as? String else {
```

Comment:
Might be cleaner to combine all of the guard lets and perform the assignments afterward